cumulusci/core/tests/test_locator_manager.py

```diff
@@ -0,0 +1,151 @@
+import unittest
+from unittest import mock
+import pytest
+
+from cumulusci.robotframework.locator_manager import (
+    register_locators,
+    translate_locator,
+    LOCATORS,
+    locate_element,
+)
+
+mock_libs = {}
+
+
+def mock_get_library_instance(name):
+    mock_libs[name] = mock.Mock(name=name)
+    return mock_libs[name]
+
+
+class TestTranslateLocator(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        LOCATORS.clear()
+        register_locators(
+            prefix="test",
+            locators={
+                "foo": {
+                    "message": "{} {}",
+                    "bar": {
+                        "baz": "//div[@class='baz']",
+                        "hello": "//span[text()='Hello, {}']",
+                    },
+                }
+            },
+        )
```

Comment:
These test classes that register locators in setUpClass should probably also have a tearDownClass that clears LOCATORS, lest we try to add new tests that use the global locators in the future and need to track down where the pollution is coming from.

Personally I would consider using a class-scoped pytest fixture instead: https://computableverse.com/blog/pytest-sharing-class-fixtures

Reply:
I added a tearDownClass method in the interest of time, since I'm unfamiliar with pytest fixtures. Thanks for the link, though. I may switch to using a fixture once I've played around with them a bit.